src/Integration.Vsix/SonarLintTagger/IssuesSnapshot.cs

```diff
@@ -125,6 +135,29 @@ public override bool TryGetValue(int index, string keyName, out object content)
             }
         }
 
+        internal /* for testing */ __VSERRORCATEGORY ToVsErrorCategory(Sonarlint.Issue.Types.Severity daemonSeverity)
```

Comment:
Logically the same as the code in `RuleSetGenerator`, except that source and target types are different.